.circleci/config.yml

```diff
@@ -39,6 +39,11 @@ jobs:
           key: v2-dependencies-{{ checksum "lois.gemspec" }}
 
       # run tests!
+      - run:
```

Comment:
This block needs to move to after we install the gem with `bin/rake install`.  Can you move this block after the `Reek` one instead?